source/server/filter_chain_factory_context_callback.h

```diff
@@ -21,7 +21,7 @@ class FilterChainFactoryContextCreator {
    * Generate the filter chain factory context from proto. Note the caller does not own the filter
    * chain context.
```

Comment:
Quick drive-by comment: IIUC, the caller does own the filter chain context now, is that right?